### DISCUSSION TITLE
Is there a way to view all codespaces created across a Github Organization?

### DISCUSSION THREAD

**Discussion by weiklr** [community/community#16349] [Codespaces]
Hi,

We are using codespaces for our github organization profile, and was wondering if there's a way for us to view all codespaces created by our devs across the different github repos within our Github Organization account?

**Comment by tino** [comment]
The only thing I found you can do is enlist them with the API, or with `gh codespace list --org ORGANIZATION`. Doesn't show you anything interesting like size or cost though...

**Comment by daphnecolson** [comment]
Hey weiklr! While there's no one-click view yet, the GitHub CLI can be your friend. Check out the `gh codespace list` command with organization filtering. It might be the solution you're looking for!
